src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,43 @@
 package mate.jdbc;
 
+import java.util.ArrayList;
+import java.util.List;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+import mate.jdbc.service.CarService;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+    private static Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        CarService carService = (CarService) injector.getInstance(CarService.class);
+        DriverService driverService = (DriverService) injector.getInstance(DriverService.class);
+        ManufacturerService manufacturerService
+                = (ManufacturerService) injector.getInstance(ManufacturerService.class);
+        List<Driver> drivers = new ArrayList<>();
+        drivers.add(driverService.get(2L));
+        Car bmwX5 = new Car("BMW X5", manufacturerService.get(1L), drivers);
+        bmwX5.setId(20L);
+        carService.create(bmwX5);
```

Comment:
It's better to make the solution more universal without using indexes explicitly. Smth like this:
``` Java
        Driver driverBob = new Driver();
        driverBob.setName("Bob");
        driverBob.setLicenseNumber("1245");
        driverService.create(driverBob);
        System.out.println(driverService.get(driverBob.getId()));
        driverBob.setName("New Bob");
        driverService.update(driverBob);
        System.out.println(driverService.get(driverBob.getId()));
```